dashboards/google-backupdr/metadata.yaml

```diff
@@ -0,0 +1,41 @@
+sample_dashboards:
+  -
+    category: Backup and DR
```

Comment:
We only need one entry per _dashboard_ here, not one per _chart_. This is more or less the same content in the readme file, but in a structured format.